### PR TITLE
Create tests for gh-emoji. Use of some more ES6 goodness :)

### PR DESCRIPTION
**Pull request by rpunkfu**

### CODE REVIEW

**Comment by zzarcon** [comment]
@oskarcieslik Thank you very much for the PR! looks quite better now ^^. Actually I'm looking for other maintainer of the repo, would you like to be part of it? I can grant you access and we can build some new features if you are interested. Think about it and let me know :smile:

**Comment by rpunkfu** [comment]
Sure, I would love to help! :) I'm currently working on demo with autocomplete :)

Also I'm a little bit worried about those hardcoded values of github emoji images with v6 at the end, it will break after they update their api..

**Comment by zzarcon** [comment]
@oskarcieslik amazing! Now you are a official collaborator of `gh-emoji` ^^. What do you mean with the `v6`??

File: test.js [hunk 1]
```diff
@@ -1,46 +1,63 @@
...
 
 test('parse', assert => {
-  assert.pass(parse(':bicyclist:') === '<img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f6b4.png?v6" class="gh-emoji gh-emoji-bicyclist" alt="bicyclist" />');
-  assert.pass(parse('foo :barber: bar :bell:').indexOf("https://assets-cdn.github.com/images/icons/emoji/unicode/1f514.png?v6") > -1);
-  assert.pass(parse('foo :barber: bar :bell:').indexOf("https://assets-cdn.github.com/images/icons/emoji/unicode/1f488.png?v6") > -1);
+  assert.ok(parse(':bicyclist:') === '<img src="https://assets-cdn.github.com/images/icons/emoji/unicode/1f6b4.png?v6" class="gh-emoji gh-emoji-bicyclist" alt="bicyclist" />',
+    'string is properly parsed'
+  );
+  assert.ok(parse('foo :barber: bar :bell:').includes(
+    'https://assets-cdn.github.com/images/icons/emoji/unicode/1f514.png?v6'),
```
Comment (rpunkfu) [inline test.js:55]: Here we're testing if parsed string includes this url, this test will break if they change v6 to something else, e.g. v7 :)

Comment (zzarcon) [inline test.js:55]: I see, you're right... maybe we can just get rid of the `?v6` part. I think it will be safer

Comment (rpunkfu) [inline test.js:55]: Yeah, I didn't think of that :p

**Pull request merged**